src/planscape/planning/tests/test_planningarea_views.py

```diff
@@ -0,0 +1,917 @@
+import json
+from unittest import mock
+from django.db import connection
+from django.contrib.auth.models import User
+from django.contrib.gis.geos import GEOSGeometry
+from django.urls import reverse
+from rest_framework_simplejwt.tokens import RefreshToken
+from rest_framework.test import APITransactionTestCase
+from planning.models import PlanningArea, Scenario, ScenarioResult
+from planning.tests.helpers import _create_planning_area, _create_scenario
+
+# Yes, we are pulling in an internal just for testing that a geometry write happened.
+from planning.views import _convert_polygon_to_multipolygon
+
+# TODO: Add tests to ensure that users can't have planning areas with the same
+# name in the same region, and that users can't have scenarios with the same
+# name in the same planning area.
+
+
+class CreatePlanningAreaTest(APITransactionTestCase):
+    def setUp(self):
+        self.user = User.objects.create(username="testuser")
+        self.user.set_password("12345")
+        self.user.save()
+
+        self.token = RefreshToken.for_user(self.user).access_token
+        self.geometry = {
```

Comment:
maybe this stuff can also go into the `helpers` file? Either this method or the geometry ?